techniques/general/G94.html

```diff
@@ -135,38 +128,35 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
       <li><a href="../html/H53">H53: Using the body of the object element</a></li>
       <li><a href="../html/H86">H86: Providing text alternatives for emojis, emoticons, ASCII art, and leetspeak</a></li>
    </ul></section>
-<section id="tests"><h2>Tests</h2>
-      <section class="procedure" id="procedure"><h3>Procedure</h3>
+<section id="tests"><h2>検証</h2>
+      <section class="procedure" id="procedure"><h3>手順</h3>
          <ol>
-            <li>Remove, hide, or mask the non-text content</li>
-            <li>Replace it with the text alternative</li>
-            <li>Check that nothing is lost (the purpose of the non-text content is met by the text alternative)</li>
-            <li>If the non-text content contains words that are important to understanding the content, the words are included in the text alternative</li>
+            <li>非テキストコンテンツを削除する、非表示にする、又は隠す。</li>
+            <li>テキストによる代替で置き換える。</li>
+            <li>何も情報が失われていないことを確認する (非テキストコンテンツの目的がテキストによる代替によって満たされている)。</li>
+            <li>非テキストコンテンツがコンテンツを理解する上で重要な単語を含む場合、その単語をテキストによる代替に含めている。</li>
          </ol>
       </section>
-      <section class="results" id="expected-results"><h3>Expected Results</h3>
+      <section class="results" id="expected-results"><h3>期待される結果</h3>
          <ul>
-            <li>Check #3 is true. If the non-text content contains words that are important to understanding the content, check #4 is also true</li>
+            <li>3 の結果が真である。非テキストコンテンツにコンテンツを理解するために必要な単語が含まれている場合は、4 の結果も真であることをチェック確認する。</li>
```

Comment:
「3 の結果が真である」という言い回しを「チェック 3 が真である」としました。合わせて、訳を調整しました。
```suggestion
            <li>チェック 3 が真である。非テキストコンテンツにコンテンツを理解するために必要な単語が含まれている場合は、チェック 4 も真である。</li>
```